Add self-serve private endpoints for Databricks on AWS (#9855)
fishtown-analytics/docs.getdbt.com: website/docs/docs/platform/secure/private-connectivity/aws/databricks.md

   - **Connection type** (for example, Databricks)
   - **URL**
   - **Connectivity status** (for example, **Success** or **Unknown**)
   - **Connections** — the number of <Constant name="dbt_platform" /> connections using the endpoint

   You can search by **Name** or **URL**.

   <Lightbox src="/img/docs/dbt-platform/private-endpoint-page.png" title="Private endpoints table showing existing endpoints, connectivity status, and the Request new button"/>

3. To request a new endpoint, click **Request new**.
4. Under **Provider type**, select **Databricks**.
5. In **Step 1: Enter your workspace name**, paste the full workspace URL (for example, `my-workspace.cloud.databricks.com`) or just the workspace name (for example, `my-workspace`).
6. In **Step 2: Select your AWS region**, choose the AWS region where your Databricks workspace is hosted. Only regions that support Databricks private connectivity are listed.
7. Click **Submit request**.

   <Lightbox src="/img/docs/dbt-platform/databricks-private-endpoint-request.png" title="Databricks endpoint request form showing Provider type, workspace name, and AWS region fields"/>

   :::warning Workspace name and region can't be changed

>    The workspace name and AWS region are fixed once the endpoint is created &mdash; you can't edit them afterward. If either value is entered incorrectly, provisioning fails. To fix it, delete the failed request and submit a new one with the correct details. No support ticket is needed.

Author: ```suggestion
   The workspace name and AWS region are fixed once you create an endpoint. This means you can't edit them afterward. If either value is entered incorrectly, provisioning will fail. To fix it, delete the failed request and create a new one with the correct details. No support ticket is needed.
```
